apps/backend/src/services/polling_service.py

```
@@ -117,122 +130,169 @@ async def _manage_polling_tasks(self, devices: List[Device]) -> None:
         # Stop polling for devices no longer in the list
         to_stop = running_device_ids - current_device_ids
         for device_id in to_stop:
-            task = self.polling_tasks.pop(device_id)
-            if not task.done():
-                task.cancel()
-                try:
-                    await task
-                except asyncio.CancelledError:
-                    pass
+            tasks = self.polling_tasks.pop(device_id)
+            for task_type, task in tasks.items():
```

Ruff (0.12.2)

Loop control variable `task_type` not used within loop body

Rename unused `task_type` to `_task_type`

(B007)